anonymizer/anonymizer.c

```diff
@@ -90,37 +90,34 @@ TRAP_DEFAULT_SIGNAL_HANDLER(stop = 1);
  */
 inline uint32_t hash_div8(const char *key, int32_t key_size)
 {
-    uint32_t c1 = 5333;
-    uint32_t c2 = 7177;
-    uint32_t r1 = 19;
-    uint32_t m1 = 11117;
-    uint32_t n1 = 14011;
-    uint64_t h = 42;
-    uint64_t * k_ptr = (uint64_t *) key;
-    uint64_t k;
-    uint32_t rep = key_size / 8;
-    uint32_t i;
-
-    for (i = 0; i < rep; i++)
-    {
-        k = *(k_ptr + i);
-        k *= c1;
-        k = ROTL64(k, r1);
-        k *= c2;
-
-        h ^= k;
-
-        h = ROTL64(h, r1);
-        h = h * m1 + n1;
-    }
-
-    h ^= h >> 33;
-    h *= 0xff51afd7ed558ccd;
-    h ^= h >> 33;
-    h *= 0xc4ceb9fe1a85ec53;
-    h ^= h >> 33;
-
-    return (uint32_t) h;
+   uint32_t c1 = 5333;
+   uint32_t c2 = 7177;
+   uint32_t r1 = 19;
+   uint32_t m1 = 11117;
+   uint32_t n1 = 14011;
+   uint64_t h = 42;
+   uint64_t * k_ptr = (uint64_t *) key;
```

Comment:
This is against coding style, it should be:
```
uint64_t *k_ptr = ...
```
If it was already reformatted, fix these issues too, please.